Tasks/NugetPackager/NuGetPackager.ps1

```diff
@@ -44,6 +44,11 @@ if ($b_versionByBuild)
     Write-Verbose "Autoversion: Getting version number from build"
     ##Get Version from Build
     
+    if($Env:SYSTEM_HOSTTYPE -eq "release") {
+        Write-Error (Get-LocalizedString -Key "Autoversion: Getting version number from build option is not supported in releases")
```

Comment:
Note we have a requirement for tasks to be usable in build and release.  What's the root reason why it's not?